packages/dds/merge-tree/src/client.ts

```diff
@@ -477,11 +483,21 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 		);
 		const op = opArgs.op;
 		const clientArgs = this.getClientSequenceArgs(opArgs);
+		let start: number | InteriorSequencePlace;
+		let end: number | InteriorSequencePlace;
+		// Kludge: using this for the error handling,
+		// even though the returned value is not used when sidedness is enabled.
 		const range = this.getValidOpRange(op, clientArgs);
-
+		if (this._mergeTree.options?.mergeTreeEnableSidedObliterate) {
+			start = { pos: op.pos1 ?? 0, side: op.before1 ? Side.Before : Side.After };
```

Comment:
when will pos1 or pos2 be undefined? Is this a valid case, or should we make it an error?